### ISSUE TITLE
There seems to be a limit to number of events in a series

### ISSUE THREAD

**Issue by Crispy66** [NPBruce/valkyrie#695]
Running into a problem with an extended event list (essentially a set of moves for the unit if tiles are seen or unseen).  I have a single event with a button and a series of events below that.  After 12 events, I can add no more.
I can easily solve this, but thought it best to post this (after searching for other posts about it) just in case it is in need of attention.
I suspect the real problem is that I am not a programmer, so my design is not elegant enough to avoid this rather crude solution.  Don't think I'm gonna fix that very easily; I read some of the discussions here and quickly become lost.

**Comment by NPBruce** [comment]
Can you please attach your scenario?  I couldn't produce this problem (there is no limit).

**Comment by Crispy66** [comment]
This is after I changed it to 2 separate smaller lists of events.
[Scenario4.zip](https://github.com/NPBruce/valkyrie/files/1512909/Scenario4.zip)

**Comment by Crispy66** [comment]
Wait - wrong scenario.

**Comment by Crispy66** [comment]
[Scenario7.zip](https://github.com/NPBruce/valkyrie/files/1512917/Scenario7.zip)

This is the correct scenario.

**Comment by NPBruce** [comment]
Which event?

**Comment by Crispy66** [comment]
It is "EventEdithMove" - now has been modified by adding "EdithMoveExtended"

**Comment by NPBruce** [comment]
I couldn't reproduce the problem.  I do not that you are using 1.7.0a, you can try 1.7.0b, but this shouldn't have changed.

**Comment by Crispy66** [comment]
I have noticed a few minor glitches similar to this while working on the scenario last night.  I would change the name of an event, then make other changes and notice that the name was still "event0".  ALso had some changes typed in on other items that seemed to revert after I did other things.  No real pattern, but the event string was enough to make me ask if there was a limit.  I noticed that several added events were not in the list, but they did exist and were intact in my list of events.  I had created them on the fly in the list that was not saving, so I thought that was interesting.
Then, to test it, I added one, made sure it was there, then saved the file, then went to another event, and back to the original.  The added event was not there.  This made me think it was a feature.

Knowing it is not, I wonder what is causing this.  I have had the program running on my laptop for several days, and I know that some programs need refreshing at times.  I will restart it tonight and see if that helps.
